modules/liveIntentIdSystem.js

```diff
@@ -143,17 +143,24 @@ export const liveIntentIdSubmodule = {
     // Don't do the internal ajax call, but use the composed url and fire it via PBJS ajax module
     const url = liveConnect.resolutionCallUrl();
     const result = function (callback) {
-      ajax(url, response => {
-        let responseObj = {};
-        if (response) {
-          try {
-            responseObj = JSON.parse(response);
-          } catch (error) {
-            utils.logError(error);
+      const callbacks = {
+        success: response => {
+          let responseObj = {};
+          if (response) {
+            try {
+              responseObj = JSON.parse(response);
+            } catch (error) {
+              utils.logError(error);
+            }
           }
+          callback(responseObj);
+        },
+        error: error => {
+          utils.logError(`${MODULE_NAME}: ID fetch encountered an error: `, error);
```

Comment:
@robertrmartinez is there a reason that sometimes you used `${MODULE_NAME}` and other times you just hardcoded the name? Just curious...

Reply:
Hi @smenzer 

`MODULE_NAME` is a common var adapters / modules use as to not replicate the literal string of their module name. But some do not utilize it.

So I only used `MODULE_NAME` in the adapters which actually have it already set.

Reply:
ah ok, good to know, thanks!